modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/DDMFormRenderingContext.java

```diff
@@ -48,6 +48,10 @@ public String getContainerId() {
 		return _containerId;
 	}
 
+	public long getDDMFormInstanceId() {
+		return _ddmFormInstanceId;
+	}
```

Comment:
@linolaoj, where this method is being used on this pull request? Can you please clarify?